### DISCUSSION TITLE
对通用信息抽取模型uie做精调，精调效果不升反降的问题

### DISCUSSION THREAD

**Discussion by guoxiaoyue111111** [PaddlePaddle/PaddleNLP#2607] [Q&A]
参考 https://github.com/PaddlePaddle/PaddleNLP/blob/develop/model_zoo/uie/doccano.md文档 做关系抽取的精调，优化后的效果没有基线模型uie-tiny的效果好，不知道是哪块出现了问题~
具体操作如下：1）使用doccano标注自己的数据40个样例，标注完并导出为jsonl；2）使用doccano.py处理数据，划分为训练集、验证集和测试机；3）使用finetune.py做精调，参数使用官方默认的参数。
结果：1）应该抽取的没抽取出来，2）即使抽取出来的，也不对。
麻烦可以指点一二，非常感谢~

**Comment by wawltor** [comment]
相关问题在issue上 https://github.com/PaddlePaddle/PaddleNLP/issues/2608  已做交流
